src/Infrastructure/Services/MockEmailService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+using Application.Contracts;
+
+namespace Infrastructure.Services
+{
+    public class MockEmailService : IEmailService
+    {
+        public Task SendEmailAsync(string to, string subject, string body)
+        {
+            // log the email instead
+            Console.WriteLine($"Email sent to {to} with subject {subject} and body {body}");
```

Comment:
We might want to output this elsewhere.
The logger, Debug console, filesystem.

Something for discussion.